src/main/resources/application.properties

```diff
@@ -34,6 +34,19 @@ endpoint.oracle.dsq_officer_collection=jdbc:dataSource
 query.oracle.dsq_officer_collection=resource:classpath:/sql/dsq_officers_collection.sql
 endpoint.mongodb.disqualifications_collection=mongodb:mongo?database=${ENDPOINT_MONGODB_DSQ_OFFICER_DB_NAME}&collection=${ENDPOINT_MONGODB_DSQ_OFFICER_COLLECTION_NAME}&operation=findDistinct&readPreference=${ENDPOINT_MONGODB_READ_PREFERENCE}
 
-endpoint.output=log:output
-
-results.initial.capacity=${RESULTS_INITIAL_CAPACITY}
\ No newline at end of file
+endpoint.log.output=log:output
+results.initial.capacity=${RESULTS_INITIAL_CAPACITY}
+
+endpoint.s3.upload=aws2-s3://datarec-store?autoCreateBucket=true
```

Comment:
Bucket name should be parameterised and we definitely do not want to create a new S3 bucket during application startup.